server/etcdmain/help.go

```diff
@@ -199,6 +199,10 @@ Security:
     Comma-separated whitelist of origins for CORS, or cross-origin resource sharing, (empty or * means allow all).
   --host-whitelist '*'
     Acceptable hostnames from HTTP client requests, if server is not secure (empty or * means allow all).
+  --tls-min-version 'TLS12'
+    Minimum TLS version supported for client and peer connections. Possible values: TLS12, TLS13.
+  --tls-max-version ''
+    Maximum TLS version supported for client ane peer connections. Possible values: TLS12, TLS13 (empty will be auto-populated by Go).
```

Comment:
Please update the messages as well, to keep consistent with the message in config.go